### PR TITLE
[Data Object] Default values should be also set to database on create

### PR DESCRIPTION
**Pull request by weisswurstkanone**
resolves #4846

### CODE REVIEW

File: models/DataObject/ClassDefinition/Data/Checkbox.php [hunk 1]
```diff
@@ -242,11 +244,10 @@ public function getForWebserviceExport($object, $params = [])
     /**
      * converts data to be imported via webservices
      *
-     * @deprecated
      * @param mixed $value
      * @param null|Model\DataObject\AbstractObject $object
      * @param mixed $params
-     * @param Model\Webservice\IdMapperInterface|null $idMapper
+     * @param $idMapper
```
Comment (blankse) [inline models/DataObject/ClassDefinition/Data/Checkbox.php]: This should be reverted. I think also the deprecated phpdocs and the isFilterable method.

Comment (weisswurstkanone) [reply models/DataObject/ClassDefinition/Data/Checkbox.php]: thx!

File: models/DataObject/ClassDefinition/Data/Checkbox.php [hunk 2]
```diff
@@ -226,7 +229,6 @@ public function getFromCsvImport($importValue, $object = null, $params = [])
     }
 
     /**
-     * @deprecated
```
Comment (blankse) [inline models/DataObject/ClassDefinition/Data/Checkbox.php]: This should be also reverted

File: models/DataObject/ClassDefinition/Data/Checkbox.php [hunk 3]
```diff
@@ -335,8 +337,10 @@ public function isEmpty($data)
         return $data === null;
     }
 
-    public function isFilterable(): bool
```
Comment (blankse) [inline models/DataObject/ClassDefinition/Data/Checkbox.php]: This should be also reverted

**Pull request merged**